source/form.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="ru" class="page">
+
+<head>
+  <meta charset="UTF-8">
+  <title>HTML Academy: Седона. Страница отзыва</title>
+</head>
+
+<body class="page__body inner-page">
+  <header class="page-header">
+    <a class="page-header__logo" href="/source/index.html">
+      <img src="https://via.placeholder.com/102x84" width="102" height="84" alt="Городок Седона">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button" title="Открыть меню"></button>
+      <div class="main-nav__wrapper">
+        <ul class="main-nav__list">
+          <li class="main-nav__item">
+            <a href="/source/index.html">Главная</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="/source/catalog.html">Фото и видео</a>
+          </li>
+          <li class="main-nav__item">
+            <a>Форма отзыва</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="https://htmlacademy.ru/" target="_blank">Html academy</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <main class="page-main">
+    <img src="http://via.placeholder.com/320x156" alt="">
+    <section class="feedback">
+      <h2 class="feedback__header">Оставьте свой отзыв</h2>
+      <p class="feedback__intro">Помогите нашим отелям стать лучше! Оставьте отзыв о них, а также о посещенных вами
+        достопримечательностях</p>
+      <form class="feedback__form" action="https://echo.htmlacademy.ru" method="get">
+        <fieldset class="feedback__fieldset feedback-fieldset--name fieldset">
```

Comment:
Тут и в аналогичных `fieldset`опечатка. 
```suggestion
        <fieldset class="feedback__fieldset feedback__fieldset--name fieldset">
```